### PR TITLE
Fixed the button and li issue reported in the review

### PR DESCRIPTION
**Pull request by manojmalik20**
Moved the setCustomStatusInitializationState action to the redux repo
Updated some snapshots and deleted some tests

#### Related Pull Requests

- Has server changes (please link here)
- Has redux changes (please link here)
- Has mobile changes (please link here)

### CODE REVIEW

File: utils/custom_status.ts [hunk 1]
```diff
@@ -3,18 +3,14 @@
...
 
 function showCustomStatusPulsatingDotAndPostHeader(state: GlobalState) {
     const preferences = getMyPreferences(state);
     const key = getPreferenceKey(Preferences.CATEGORY_CUSTOM_STATUS, Preferences.NAME_CUSTOM_STATUS_TUTORIAL_STATE);
-    if (!(preferences[key] && preferences[key].value === Preferences.CUSTOM_STATUS_MODAL_VIEWED)) {
-        return true;
-    }
-
-    return false;
+    return !(preferences[key] && preferences[key].value === Preferences.CUSTOM_STATUS_MODAL_VIEWED);
```
Comment (chetanyakan) [inline utils/custom_status.ts:13]:
```suggestion
    const hasViewedCustomStatusModal = preferences[key] && preferences[key].value === Preferences.CUSTOM_STATUS_MODAL_VIEWED;
    return !hasViewedCustomStatusModal;
```

File: utils/custom_status.ts [hunk 2]
```diff
@@ -3,18 +3,14 @@
 import {getMyPreferences} from 'mattermost-redux/selectors/entities/preferences';
 
 import {getPreferenceKey} from 'mattermost-redux/utils/preference_utils';
+import {Preferences} from 'mattermost-redux/constants';
 
 import {GlobalState} from 'types/store';
-import {Preferences} from 'utils/constants';
 
 function showCustomStatusPulsatingDotAndPostHeader(state: GlobalState) {
```
Comment (chetanyakan) [inline utils/custom_status.ts:10]: Should we move these utils to selectors?

**Review by chetanyakan — APPROVED** [review]

File: utils/custom_status.ts (hunk 2 above)
Comment (manojmalik20) [reply utils/custom_status.ts:10]: We can but I think selectors are only used to get something from state and return it... If we move these to selectors then meaning of selectors will be changed.

Comment (chetanyakan) [reply utils/custom_status.ts:10]: Update this as discussed.

**Pull request merged**